packages/lit-dev-content/site/tutorials/content/intro-to-lit/01.md

```diff
@@ -36,3 +36,12 @@ In Lit, most things start with defining a _component_. Here we've given you a st
     The `render()` method defines your component's internal DOM. The `html` tag function processes a template literal and returns a `TemplateResult`—an object that describes the HTML for Lit to render. Every Lit component should include a `render()` method.
 
 You should see the greeting message in the **Result** pane.
+
+{% aside "info" %}
+
+Decorators.
+
+The TypeScript version of this code uses the `@customElement` decorator to register your class with the browser as a new element. The JavaScript version uses `customElements.define()` to do the same thing. This is a matter of style. You can use decorators—or not use decorators—in either language. For more information, see [Decorators](/docs/components/decorators/).
+
+{% endaside %}
```

Comment:
is it worth mentioning the TS/JS toggle. Surprising amount of people online don't notice that it exists and have issues with some of our decorators. Specifically `@query`

Reply:
ah i see you talk about query a later step 👍 👍

Reply:
Yeah, I talk about that but mentioning the toggle explicitly is a good idea. It was added after the tutorial was written, so there were a number of confusing bits.

Reply:
Added note, with a little extra switchable sample as a demo.